src/opm/parser/eclipse/Parser/ParseContext.cpp

```diff
@@ -140,14 +140,21 @@ namespace Opm {
         this->ignore_keywords.insert(keyword);
     }
 
+    void ParseContext::handleError(const std::string& errorKey,
+                                   const std::string& msg_fmt,
+                                   const std::optional<KeywordLocation>& location,
+                                   ErrorGuard& errors) const {
+        InputError::Action action = this->get( errorKey );
```

Comment:
How is this supposed to work?  In a `static` member function there is no `this` object.

Reply:
You are looking at the old non-static method which now uses `this->` to extract a value and then subsequently call the new `static` method.